components/automate-chef-io/content/docs/install.md

```diff
@@ -119,3 +126,126 @@ sudo chef-automate config patch config.toml
 ```
 
 and retry opening Chef Automate in your browser.
+
+### Configuring External Data Stores
```

Comment:
I thought it was worth spelling this out here (instead of pointing to the configuration page) because if someone knows upfront they need an external data store, we want them to do their initial deploy that way. I'm not sure this is the best way to encourage that, so I'm willing to hear other ideas.

Reply:
I'm wondering if we want the the initial install instructions to be the _only_ instructions for now?

Reply:
@stevendanna That makes sense to me, so I removed the ones on the configuration page.